frontend/src/scenes/session-recordings/player/share/PlayerShare.tsx

```diff
@@ -29,7 +29,7 @@ export function PlayerShareRecording(props: PlayerShareLogicProps): JSX.Element
                 fullWidth
                 center
                 sideIcon={<IconCopy />}
-                onClick={() => void copyToClipboard(url, 'recording link').then(captureException)}
+                onClick={() => void copyToClipboard(url, url).catch((e) => posthog.captureException(e))}
```

Comment:
i'll leave this in and preserve my shame
it's a valid change but will be in https://github.com/PostHog/posthog/pull/28124 🙈

Reply:
Thank you for the conflicts, so very kind

Reply:
🙈 :moanna-you-are-welcome-dot-meme: